aws-rds-dbclusterparametergroup/src/test/java/software/amazon/rds/dbclusterparametergroup/UpdateHandlerTest.java

```diff
@@ -105,156 +98,34 @@ public void setup() {
                 .build();
 
         requestSameParams = ResourceHandlerRequest.<ResourceModel>builder()
-            .clientRequestToken("token")
-            .desiredResourceState(RESOURCE_MODEL)
-            .previousResourceState(RESOURCE_MODEL)
-            .desiredResourceTags(translateTagsToMap(TAG_SET))
-            .logicalResourceIdentifier("logicalId").build();
+                .clientRequestToken("token")
+                .desiredResourceState(RESOURCE_MODEL)
+                .previousResourceState(RESOURCE_MODEL)
+                .desiredResourceTags(translateTagsToMap(TAG_SET))
+                .logicalResourceIdentifier("logicalId").build();
         requestUpdParams = ResourceHandlerRequest.<ResourceModel>builder()
-            .clientRequestToken("token")
-            .desiredResourceState(RESOURCE_MODEL)
-            .previousResourceState(RESOURCE_MODEL_PREV)
-            .desiredResourceTags(translateTagsToMap(TAG_SET))
-            .logicalResourceIdentifier("logicalId").build();
+                .clientRequestToken("token")
+                .desiredResourceState(RESOURCE_MODEL)
+                .previousResourceState(RESOURCE_MODEL_PREV)
+                .desiredResourceTags(translateTagsToMap(TAG_SET))
+                .logicalResourceIdentifier("logicalId").build();
```

Comment:
I understand it is quite some copy-pasting, but I'd suggest placing request building logic in the corresponding tests.